### ISSUE TITLE
Add Significant Digits

### ISSUE THREAD

**Issue by Conless** [stargazerZJ/yexi#1]
You may consider the requirement of significant digits in calculation, or your experiment report would be "fang xue" by yexi 🥵

**Comment by stargazerZJ** [comment]
You are right, but I think that rules concerning significant digits are redundant when we calculate in fp64.
I hope the grader agrees.
PR welcome
